### ISSUE TITLE
New Replies for by the bot.

### ISSUE THREAD

**Issue by mrlancelot** [mrlancelot/twitter-bot-generic#1]
For now, the Twitter bot uses the values from a JSON file while replying. Adding a way to give witty replies without the use of a database.

**Comment by karmanya007** [comment]
Does the replies currently come from animals.json file??

**Comment by mrlancelot** [comment]
@karmanya007 Yes its animals.json file

**Comment by karmanya007** [comment]
So you want to create another JSON file for different replies or add to this one?

**Comment by mrlancelot** [comment]
@karmanya007 Yes for now more replies.

**Comment by karmanya007** [comment]
Yes for a new JSON file? And what kind of replies are you expecting?

**Comment by asfo** [comment]
@mrlancelot maybe, you should create a new issue so the JSON doesn't accept single comments from an animal, and instead a lot of facts. Then open a new one and implement the logic to connect to a DB (could be for example an SQLite so it can be moved) and there store the facts. So this could be a reusable base code for other twitter bot projects.

**Comment by karmanya007** [comment]
@mrlancelot Could you be a little more specific in terms of what kind of replies are you expecting so that I can start working on this feature.
